src/Bookings.js

```diff
@@ -1,19 +1,77 @@
 import React, { Component } from "react";
 import Search from "./Search.js";
-// import SearchResults from './SearchResults.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import SearchResults from "./SearchResults.js";
 
 export default class Bookings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      FakeBookings: null,
```

Comment:
This is a tiny nitpick, but generally we prefer camelCase variable naming for state variables.